justfile-rust

```diff
@@ -1,12 +1,15 @@
-# This justfile includes recipes for building and packaging the validator for
-# release. This file is separated so that we can invoke cargo, etc when building
-# defaults. If this logic were in the top-level justfile, then these tools would
-# be invoked (possibly updating Rust, etc), for unrelated targets.
+# This justfile includes recipes for building and packaging the validator and
+# reinitialize-pods crates for release. This file is separated so that we can
+# invoke cargo, etc when building defaults. If this logic were in the main
+# justfile, then these tools would be invoked (possibly updating Rust, etc),
+# for unrelated targets.
 #
-# Users are expected to interact with this via the top-level Justfile.
+# Users are expected to interact with this via the main Justfile.
+
+crate := env_var('TARGETCRATE')
```

Comment:
should we have a default here?

Reply:
I'd like the env var to be required, so this would fail with a clear error if the var is not explicitly set.